chapters/conclusion.tex

```diff
@@ -1,4 +1,4 @@
-We hope you have enjoyed \textit{Data for Development Impact: The DIME Analytics Resource Guide}.
+We hope you have enjoyed \textit{Development Research in Practice: The DIME Analytics Data Handbook}.
```

Comment:
in the intro we used only Development Research in Practice, no subtitle. Let's be consistent. but i think the full title is probably best.

Reply:
c3fef57e228e5e73bc427e73a8bdf19bae1a5881